### ISSUE TITLE
.article bundling doesn't work

### ISSUE THREAD

**Issue by ghost** [StudioMercury/digital-publishing-tools-for-wordpress#28]
Dear @StudioMercury,

we have successfully connected your plugin to DPS. When previewing articles in the browser from Wordpress or in your plugin everything looks fine (CSS and fonts are loaded). Also the cloud connection works well (we can push articles to the cloud and make changes from the plugin).

However apparently the bundling doesn't work as intended: Within the DPS app the content and meta data are displayed, but our css and font files are not loading. It might have something to do with this issue: #24 

We are storing the files in our theme as suggested within the folder publish-templates/HTMLResources/... and have included them to the bundling code in the template file as suggested using this code:

```
<?php
    if( isset($_GET["bundlr"]) ) {
        $filePath = 'HTMLResources/';
        $urlPath = 'navto://';
    } else {
         $filePath = get_bloginfo('template_directory') . '/publish-templates/HTMLResources/';
    }

add_filter('dpsfa_bundle_article', function($entity){
        $path = pathinfo($entity->template);
        return array(
            "HTMLResources/css/app.css" => $path['dirname'] . '/HTMLResources/css/app.css',
               );
    }); 
?>
```

we are calling the files with this code:

```
<link rel="stylesheet" href="<?php echo $filePath; ?>css/cookionista_app.css">
```

Are we doing something wrong? Do you have any idea of what could be a possible reason for this issue?

Thank you for your help. We really appreciate your plugin and support!

**Comment by josedesousa** [comment]
Hello,

I was having the same issue, it seems the last commit 99233ab upset the process. I reverted the build back to the previous version and the bundling is working once more.

**Comment by ghost** [comment]
@josedesousa-tcc Thanks a lot Jose! That solved the problem for us too...
@StudioMercury Seems like there really is something upsetting the bundling process in the last commit...

**Comment by StudioMercury** [comment]
@josedesousa-tcc @ju21 
We just released an update 2.0.10 with an improvement to the packaging of the article. We updated the way the template. If you look inside the article.php file you'll see we're looking for full paths (or relative paths to the server now). This is part of a larger goal to allow any template (or theme) to be used for creating an article. Relative paths are no longer relative to the article. You can manually specify relative paths in the filter (dpsfa_bundle_article). It's been committed to github but will be available via wordpress's plugin portal within the hour.
